src/css/home.scss

```diff
@@ -72,12 +72,22 @@
       margin-top: 64px;
       margin-bottom: 72px;
 
+      ul {
```

Comment:
@astanovferuz , for the spacing between `cta-links` and `ticker`, it can be change something styles in `ticker` class for mobile screen as following:
```
.ticker {
   ...
   margin-left: -15px;
   margin-right: -15px;
  @media (min-width: #{$bp-medium}) {
      position: absolute !important;
      left: 0;
      right: 0;
      bottom: 0;
  }
}
```